data/hai/unfettered side missions.txt

```diff
@@ -65,3 +65,485 @@ mission "Unfettered returning home"
+			choice
+				`	(Go back to the shipyard.)`
+					goto leave
+				`	"Why would they mind a fleet or a different ship?"`
+			`	"Coming from a human, arriving with a fleet would be nothing less than an invitation to fight," says the first. "However, as this ship is ancient they will try to preserve it," the other one adds.`
+			label leave
+			`	They walk away until that you cannot hear them speak anymore, leaving you with the access codes to the ship so you can bring it to the shipyard. With the aid of the autopilot, you manage to bring it without any complications.`
+				accept
+			label foolish
+			`	You get your weapon out and proceed to shoot on the ship.`
+			branch miss
+				random < 50
+			branch lucky?
+				random < 2
+			`	Your shot reaches the ship, but the small weapon has no chance of doing anything to the heavy armor, even if it is somewhat worn. The ship then shoots back at you, barely missing and causing a geyser of sand to erupt at you and force you to drop your weapon. One of the two Hai then approaches you, seemingly disappointed.`
+			`	"You're lucky to still be alive, but even so you are of no use to us if you think it wise to use your firearm on a spaceship. Farewell." They turn and leave without another word, and the ship flies off.`
+				decline
+			label lucky?
+			`	You manage to land a shot, although it seems to do nothing to the large spacecraft before you. Then you hear a small explosion. You aren't sure whether the damaged ship's systems failed on their own, or if by some stroke of luck, your shot did something to it.`
+			`	Unfortunately, you do not get a chance to ponder it, as the ship abruptly explodes, taking you and everyone else on the beach down with it.`
+				die
+			label miss
+			`	You miss your shot. Neither the ship nor the two Hai react. You feel like you heard a chuckle, but you're not sure where it came from.`
+				goto "try again"
+	on defer
+		set "Hai Legacy 1: deferred"
+	on accept
+		clear "unfettered: ocean dialog"
+		clear "unfettered: pilot dialog"
+		log "Accepted a somewhat shady deal for an old Hai ship, an Anomalocaris, that seems to still be in good functioning state."
+
+
+
+mission "Hai Legacy 2"
+	autosave
+	landing
+	name "Heritage Reclaimed"
+	description "Use the newly acquired Anomalocaris to bring aid to the Unfettered Hai on <planet> by <date>, with it as your flagship and preferably without any other ships."
+	clearance
+	# Deadline to make sure the player doesn't gain rep for free.
+	deadline 30
+	cargo "food (aid)" 121
+	source "Hai-home"
+	destination "Firelode"
+	to offer
+		not "Hai Legacy 1: declined"
+		has "First Contact: Unfettered: offered"
+		"reputation: Hai (Unfettered)" < 0
+		not "Unfettered Jump Drive 1: offered"
+		"credits" >= 10000000
+	to accept
+		has "ship model: Anomalocaris"
```

Comment:
Technically, wouldn't the cargo be loaded on a different ship if the Anomalocaris isn't initially your flagship? And if that's the case, and you're told not to bring any other ships, wouldn't that result in error?

I think it might be better to have the mission have a block that says you can't accept it unless the Anomalocaris is your flagship, and change this to be "has: "flagship: Anomalocaris" or however that is done.

Reply:
well actually the cargo is automatically distributed to non flagships, and when you park other ships their cargo will be transfered

and that amount of cargo fits perfectly on this ship, so if the player doesn't do weird things like already having stuff in cargo and selling a ship from the shipyard (in which case the mission wont trigger until there is enough cargo space available), it should be fine

Reply:
If the mission won't trigger if there's enough cargo space, there should probably be a block message. This is a follow-up directly trailing off of the first mission; it would be odd if it just didn't appear/offer without saying anything.

Reply:
actually I cant have a blocked dialog because of the way it works rn (the mission is pre-activated but cant be accepted until you come out of the shipyard with the ship)